libraries/tools/kotlin-gradle-plugin/src/common/kotlin/org/jetbrains/kotlin/gradle/targets/native/tasks/KotlinNativeTasks.kt

```diff
@@ -796,7 +796,7 @@ internal class CacheBuilder(
     private val partialLinkageMode: String
         get() = settings.toolOptions.freeCompilerArgs.get().mapNotNull { arg ->
             arg.substringAfter("$PARTIAL_LINKAGE_PARAMETER=", missingDelimiterValue = "").takeIf(String::isNotEmpty)
-        }.lastOrNull() ?: PartialLinkageMode.DEFAULT.name
+        }.lastOrNull() ?: PartialLinkageMode.DEFAULT.name.toLowerCase()
```

Comment:
```suggestion
        }.lastOrNull() ?: PartialLinkageMode.DEFAULT.name.lowercase()
```
Want to avoid the use of the implicit host JVM's default `Locale` and its casing rules, right?

Reply:
Yes you are right - thank you for this :D